### PR TITLE
haproxy: add lua support

### PR DESCRIPTION
**Pull request by abhay-s-rawat**
Added lua support for haproxy formula

### CODE REVIEW

**Comment by carlocab** [comment]
Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

File: .github/.DS_Store
Comment (carlocab) [inline .github/.DS_Store:1]: Please remove this.

**Review by carlocab — CHANGES_REQUESTED** [review]
I've already said this earlier, but I'd like to highlight the following part of my previous comment, particularly the part in bold:

> Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:
> 
> > - **One formula per commit; one commit per formula.**
> > - Keep merge commits out of the pull request.

**Comment by abhay-s-rawat** [comment]
I have no Idea what is the problem here. Anyways I will not submit pr sorry for inconvenience.

**Pull request closed without merging**